apps/mark-scan/backend/src/app.ts

```diff
@@ -180,16 +179,7 @@ export function buildApi(
     },
 
     setAcceptingPaperState(): void {
-      if (
-        isFeatureFlagEnabled(
-          BooleanEnvironmentVariableName.USE_MOCK_PAPER_HANDLER
-        )
-      ) {
-        return;
-      }
-
```

Comment:
I didn't understand why this was here. I needed to remove it because I wanted to use the mock paper handler flag in tests to use the printing --> straight to interpretation mocked behavior, but I needed to advance the state machine into a `waiting_on_ballot_data` state to print at all.